tools/cold-store/README.md

```diff
@@ -0,0 +1,100 @@
+# Cold Storage testing tool
+
+## Workflow
+
+Start by trying something on localnet,
+then move on to test you code on a dedicated machine
+with real archival data.  
+Ideally, on archival machine we only need to do every step before experimenting once.
+But accidents happen, and we should be mindful of the time it takes us
+to recover from them.
+
+### Localnet
+
+#### Prepare data
```

Comment:
Text wrapping in this file is sub-optimal.

Reply:
I will work with this file a lot in the nearest future. We will have some time to discuss proper way of writing it before the file is stabilised.
I think that rendered file look OK and I do not like to write long lines in .md file. And like blank lines. Personal preference that I am willing to discuss. But right now I will merge it as is.